### PR TITLE
Updated Examples and Changelog in Readme.md

### PR DESCRIPTION
**Pull request by kunaldhand**
Updated Changelog 
from https://github.com/noplay/python-mysql-replication/blob/master/CHANGELOG
to https://github.com/noplay/python-mysql-replication/blob/main/CHANGELOG

Updated Examples block
from
```
from pymysqlreplication import BinLogStreamReader
mysql_settings = {'host': '127.0.0.1', 'port': 3306, 'user': 'root', 'passwd': ''}

stream = BinLogStreamReader(connection_settings = mysql_settings, server_id=100)

for binlogevent in stream:
    binlogevent.dump()

stream.close()
```

to

```
 from pymysqlreplication import BinLogStreamReader

MYSQL_SETTINGS = {"host": "127.0.0.1", "port": 3306, "user": "root", "passwd": ""}


def main():
    stream = BinLogStreamReader(
    connection_settings=MYSQL_SETTINGS, server_id=3, blocking=True
    )

    for binlogevent in stream:
        binlogevent.dump()

    stream.close()


if __name__ == "__main__":
    main()
```

Also updated the output according to the new code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -288,7 +315,7 @@ https://python-mysql-replication.readthedocs.org/en/latest/developement.html
 
 Changelog
 ==========
-https://github.com/noplay/python-mysql-replication/blob/master/CHANGELOG
+https://github.com/noplay/python-mysql-replication/blob/main/CHANGELOG
```
Comment (sean-k1) [inline README.md:318]: cool!

File: README.md [hunk 2]
```diff
@@ -120,14 +120,22 @@ This example will dump all replication events to the console:
...
 
-stream.close()
+    for binlogevent in stream:
+        binlogevent.dump()
+
+    stream.close()
+
+
+if __name__ == "__main__":
+    main()
```
Comment (sean-k1) [inline README.md:138]: I didn't mean for you to change this area. 

I was asking to change the link on line 115.
I'm sorry I didn't convey my intentions clearly.

Comment (kunaldhand) [reply README.md]: Sorry It was my mistake. I resolved the issue now. Changed the "Example Directory" 
from https://github.com/noplay/python-mysql-replication/tree/master/examples 
to https://github.com/noplay/python-mysql-replication/tree/main/examples

additionally I roll backed the changes done to Example and Output code in example block.

**Review by sean-k1 — APPROVED** [review]
LGTM ! Thanks your hard Work!

**Pull request merged**